lib/endon.ex

```diff
@@ -272,10 +280,13 @@ defmodule Endon do
 
       ## Options
 
-        * `:order_by` - By default, orders by primary key descending
+        * `:order_by` - By default, orders by primary key ascending, and takes the last `count`
```

Comment:
I'm probably missing something here, but it reads to me like `last/2` orders by pk descending rather than ascending.
```
iex(3)> UserSingle.last(42)
["from u0 in UserSingle, order_by: [desc: u0.id], limit: ^42"]
```

Reply:
Yeah ... so this is just the "should generally work w/ any DB type way to use SQL to get the last n rows" approach.  The docs say "this thing will order by primary key ascending and take the last n" as a way of describing the effect.  What's really happening, is it does the reverse sort and takes the _first_ n, then reverses the order of the result.